services/director-v2/openapi.json

```diff
@@ -1,5 +1,5 @@
 {
-  "openapi": "3.1.0",
+  "openapi": "3.0.0",
```

Comment:
a downgrade?

Reply:
well last time I reverted it, or someone did it. I don't really know, I don't really understand the real problem behind it, I think @pcrespov knows and should be the go to person regarding this (he actually told me this trick ;) )

Reply:
OK, makes sense. Thanks for the clarification. I will revert this change then.